### PR TITLE
Update PhimmoichillProvider.kt

### PR DESCRIPTION
**Pull request by tuan041**

### CODE REVIEW

File: PhimmoichillProvider/src/main/kotlin/com/lagradost/PhimmoichillProvider.kt [hunk 1]
```diff
@@ -89,7 +89,7 @@ class PhimmoichillProvider : MainAPI() {
             .toIntOrNull()
         val tvType = if (document.select("div.latest-episode").isNotEmpty()
         ) TvType.TvSeries else TvType.Movie
-        val description = document.select("div#film-content").text().trim()
```
Comment (C10udburst) [inline PhimmoichillProvider/src/main/kotlin/com/lagradost/PhimmoichillProvider.kt:92]: this doesnt seem to work

image

File: PhimmoichillProvider/src/main/kotlin/com/lagradost/PhimmoichillProvider.kt [hunk 2]
```diff
@@ -100,7 +100,7 @@ class PhimmoichillProvider : MainAPI() {
                 val titleHeader = it.select("p") ?: return@mapNotNull null
                 val recUrl = titleHeader.attr("href") ?: return@mapNotNull null
                 val recTitle = titleHeader.text() ?: return@mapNotNull null
-                val poster = it.select("img").attr("src")
```
Comment (C10udburst) [inline PhimmoichillProvider/src/main/kotlin/com/lagradost/PhimmoichillProvider.kt:103]: these seem to be exactly the same
image

**Review by C10udburst — CHANGES_REQUESTED** [review]

File: PhimmoichillProvider/src/main/kotlin/com/lagradost/PhimmoichillProvider.kt (hunk 2 above)
Comment (tuan041) [reply PhimmoichillProvider/src/main/kotlin/com/lagradost/PhimmoichillProvider.kt:103]: Why it doesn't show the poster?

File: PhimmoichillProvider/src/main/kotlin/com/lagradost/PhimmoichillProvider.kt (hunk 1 above)
Comment (tuan041) [reply PhimmoichillProvider/src/main/kotlin/com/lagradost/PhimmoichillProvider.kt:92]: Yeah, I see

**Pull request merged**